index.html

```diff
@@ -1516,55 +1526,107 @@ <h3>Introduction</h3>
+
+					<ul>
+						<li>It MUST NOT support <a data-cite="epub-rs-33#sec-epub-rs-conf-remote-res">remote
+								resources</a> [[epub-rs-33]].</li>
+						<li>It MUST NOT resolve [=path-absolute-URL strings=] [[url]].</li>
+						<li>As remote resources, forms, and scripting are not supported, it MUST NOT allow [=eBraille
+							publications=] to have <a data-cite="epub-rs-33#sec-epub-rs-network-access">network
+								access</a> [[epub-rs-33]]. (Note that barring network access should not prevent users
+							from following <a data-cite="epub-rs-33#sec-epub-rs-external-links">external hyperlinks</a>
+							[[epub-rs-33]] in a publication.)</li>
+						<li>It MUST only support [=eBraille content documents=] in the <code>spine</code>. The reading
+							system MAY reject eBraille publications with other formats in the spine or ignore their
+							entries when rendering the publication.</li>
+						<li>It MUST NOT support <a data-cite="epub-rs-33#sec-xhtml-forms">form submission</a>
+							[[epub-rs-33]].</li>
+						<li>It MUST NOT support <a data-cite="epub-rs-33#sec-scripted-content">scripting</a>
+							[[epub-rs-33]].</li>
+						<li>It is only required to support <a data-cite="epub-rs-33#sec-ocf">OCF processing</a>
+							[[epub-33]] if it supports <a href="#ebrl-packaging">packaged eBraille
+							publications</a>.</li>
+					</ul>
+				</section>
+
+				<section id="rs-req-unsupported">
+					<h4>Unsupported features</h4>
+
+					<p>The following EPUB reading system features [[epub-rs-33]] are not supported in the authoring of
+						eBraille publications. As these features do not affect the privacy or security of eBraille
+						publications, reading systems built on EPUB rendering engines MAY support the features although
+						they SHOULD be disabled, if possible, to discourage use by eBraille creators:</p>
+
+					<ul>
+						<li><a data-cite="epub-rs-33#sec-pkg-doc-manifest">manifest fallbacks</a>.</li>
+						<li><a data-cite="epub-rs-33#sec-svg">SVG content documents</a>. (Note that this does not affect
+								<a data-cite="epub-rs-33#sec-xhtml-svg">embedded SVG</a>.)</li>
+						<li><a data-cite="epub-33#sec-css-prefixed">EPUB 3's prefixed CSS properties</a>.</li>
+						<li><a data-cite="epub-rs-33#epub-rs-33/#sec-rendering-control">fixed layouts</a>.</li>
+						<li><a data-cite="epub-rs-33#sec-epub-rs-conf-backward">backward compatibility</a> or <a
+								data-cite="epub-rs-33#sec-epub-rs-conf-forward">forward compatibility</a> with other
+							EPUB versions.</li>
+					</ul>
 
-				<p>In addition, eBraille reading systems MUST support PDF for tactile graphics. If an eBraille reading
-					system is not capable of rendering PDFs embedded in [=eBraille content documents=], it MUST provide
-					a mechanism that allows the user to open the graphics in an application that supports PDF
-					rendering.</p>
+					<p>eBraille reading systems not built on EPUB rendering engines MUST NOT add support for these
```

Comment:
Did you really mean "not build on EPUB rendering engines"?  seems if they were build on these EPUB rendering engines they must remove that support.

Reply:
I've changed the name to "Non-critical features", as "Unsupported features" is too broad. There are critical features that must not be supported in the first section. I've also tried to explain this better in the first paragraph.